tests/python/symm_JK/input.dat

```diff
@@ -0,0 +1,65 @@
+import psi4
```

Comment:
I think by default the tests are all lower case, it would be good to stick with that. (I may be wrong here, worth checking).